generators/server/templates/src/main/resources/config/liquibase/oauth_client_details.csv

```diff
@@ -0,0 +1,2 @@
+client_id;resource_ids;client_secret;scope;authorized_grant_types;authorities;access_token_validity;refresh_token_validity;additional_information;autoapprove
```

Comment:
Add the web_server_redirect_uri column so that users can easily configure it they want